cmd/harbor/root/project/create.go

```diff
@@ -45,8 +46,7 @@ func CreateProjectCommand() *cobra.Command {
 	}
 
 	flags := cmd.Flags()
```

Comment:
```suggestion
	flags := cmd.Flags()
	flags.StringVarP(&opts.ProjectName, "name", "", "", "Name of the project")

```

Reply:
What's the tradeoff over using flags instead of args?

Reply:
to have a consistent behaviour across other resources like registry.